src/libs/google-drive-uploader.js

```diff
@@ -0,0 +1,268 @@
+/**
+ * Unfortunately, Google is not providing the client for uploading files
+ * and we didn't found cool library for this feature
+ *
+ * We found it:
+ * GitHub https://github.com/tanaikech/ResumableUploadForGoogleDrive_js
+ *
+ * We can use it as it and refactor in the future
+ */
+/**
+ * ResumableUploadToGoogleDrive for Javascript library
+ * GitHub  https://github.com/tanaikech/ResumableUploadForGoogleDrive_js<br>
```

Comment:
>v1.0.0 (October 11, 2019)
>Initial release.
Wow we're so lucky the lib exists, it has been released so recently :)

Reply:
Which version/commit did you copy here?
There are 5 commits in that github repo.

The file is different from https://github.com/tanaikech/ResumableUploadForGoogleDrive_js/blob/master/resumableupload_js.js

You have added `export` in the end, created the `context` variable.

If a new version with a bug fix comes out, say `v1.0.1`, how are we going to update this file?

We could fork the repo, make the changes, add `package.json` and `npm install` it [directly from github](https://stackoverflow.com/questions/17509669/how-to-install-an-npm-package-from-github-directly) (without publishing to `npm`).

This way we would have the changes you made in a github repo's history. Also the commit you forked from would be visible in github this way.

Reply:
>> If a new version with a bug fix comes out

we don't need new version, we need restructure this code and keep it in our repo

>> how are we going to update this file

if imagine, that we need it - we can see on the diff in the new version and apply changes
but I have a huge doubt that this situation is possible

You can think that we are not using a library. Because it's a pseudo library. You can think that it's working code snippet from the internet. In fact, it's something simple between library and code snippet. The author should rework it to make it a library.

Why is this not a library in my opinion?

The author didn't take care of ways to integrate it into the project. It has not package.json with version, it can't be imported as a module. A only way to add it to the project - including it to html (as described in readme). In 2020.